src/components/About/About.tsx

```diff
@@ -1,19 +1,22 @@
 import { LinkButton } from "../LinkButton/LinkButton";
+import Image from "next/image";
 
 export const AboutSection = ({ header }: { header: { fields: any } }) => {
   const { id, title, subtitle, thumb } = header.fields;
 
   return (
     <section id={id} className="w-full bg-grey-100 py-8 lg:py-12">
       <div className="w-full max-w-[1440px] mx-auto px-4 flex flex-col gap-6">
-        <h2 className="text-[30px] leading-[36px] tracking-[-0.0075em] font-bold text-left lg:text-left text-center">
+        <h2 className="text-[30px] leading-[36px] tracking-[-0.0075em] font-bold text-left">
           {title}
         </h2>
         <div className="flex flex-col lg:flex-row gap-6">
-          <img
+          <Image
             alt=""
             src={`https:${thumb.fields.file.url}`}
-            className="w-full lg:w-[462px] lg:h-[374px] rounded-lg"
+            width={500}
+            height={374}
+            className="w-fit max-h-[374px] rounded-lg"
```

Comment:
here we actually needed the w-full and lg:w-[462px] for different screen sizes hehe